shared/src/test/scala/squants/thermal/TemperatureSpec.scala

```diff
@@ -106,52 +121,63 @@ class TemperatureSpec extends FlatSpec with Matchers {
     x.inFahrenheit should be(Fahrenheit(32))
     x.in(Kelvin) should be(Kelvin(273.15))
     x.in(Fahrenheit) should be(Fahrenheit(32))
+    (x.in(Rankine) - Rankine(491.67)).value < 0.000000000001 should be(right = true)
```

Comment:
Probably. I adopted that style for consistency with the other tests in TemperatureSpec.

I'm not familiar with the `=~` operator. Using `+-`, it would the line would look like this:

`x.in(Rankine).value should be(491.67 +- 0.000000000001)` which is less verbose than the current form, but doesn't test if the output is boxed in Rankine (it only tests the value conversion). I'm open to suggestions on how best to restructure this test, and we should probably restructure the tests in the Fahrenheit boxing tests in that file too (see https://github.com/typelevel/squants/blob/master/shared/src/test/scala/squants/thermal/TemperatureSpec.scala#L97 for example).

Overall, there's a lot of boilerplate and repeated code in the thermal package. The 9:5 scaling factor between Celsius and Fahrenheit is repeated multiple times, as are several other magic values. At some point, we should think about how to restructure the internals to reduce duplication and boilerplate.